index.html

```diff
@@ -565,7 +567,15 @@ <h2 class="contact__me__title">Contact Me</h2>
             placeholder="Write your message here ..."
             required
           ></textarea>
-          <button class="btn btn1 btn2" type="submit">Get in Touch</button>
+          <button
+            class="btn btn1 btn2"
+            id="submitButton"
+            type="submit"
+            onclick="validation()"
+          >
+            Get in Touch
+          </button>
+          <h3 class="error-message" id="text"></h3>
```

Comment:
- [ ] Your Project's been approved, is now time to merge 👯‍♀️✨💎

### Project Approved ✔✔✔
![giphy (1)](https://user-images.githubusercontent.com/54015740/151438830-60b2fdde-f770-44a5-93cc-75a5d239a582.gif)